docs/how-tos/write-did-and-query-verkey/README.md

```diff
@@ -115,7 +115,7 @@ any number of additional identities to the ledger, with just a handful of
 lines of code. Try creating some.
 
 You might try the ["Rotate a Key"](../rotate-key/README.md)
-how-to, which can be done in only one step one you complete this one.
+how-to, which can be done in only one step once you have complete this one.
```

Comment:
There's another grammar mistake on the same line: "complete" should be "completed".

Do you want to update your pull request?